.gitignore

```diff
@@ -9,3 +9,4 @@ nbproject
 .settings
 vendor
 composer.phar
+phpunit.xml.dist
```

Comment:
i had that fixed but in excitement forgot to pull when went to work computer and overwrote it :(
Good catch.